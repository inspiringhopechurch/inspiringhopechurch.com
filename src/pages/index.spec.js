import React from "react";
import { shallow } from "enzyme";
import Index from "./index";
import data from "../fixtures/data";
jest.mock("../components/map", () => () => <></>);

describe("Index", () => {
  it("should render without crashing", () => {
    const wrapper = shallow(<Index data={data.data} />);
    expect(wrapper).toMatchSnapshot();
  });
});
