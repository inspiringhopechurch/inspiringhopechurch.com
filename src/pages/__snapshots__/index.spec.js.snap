// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Index should render without crashing 1`] = `
<Fragment>
   
  <SEO
    article={false}
    banner={null}
    desc={null}
    pathname={null}
    title="Inspiring Hope Church"
  />
  <section
    className="index-page fade-in hero is-large"
  >
    <div
      className="hero-body"
    >
      <div
        className="container"
      >
        <div
          className="columns is-vcentered"
        >
          <div
            className="column"
          >
            <h1
              className="has-text-white"
            />
          </div>
        </div>
      </div>
    </div>
    <div
      className="hero-foot"
    >
      <nav
        className="tabs is-centered"
      >
        <div
          className="container"
        >
          <ul>
            <li
              className="pr-6"
            >
              <mockConstructor
                className="button is-white is-size-4-tablet"
                to="/events/easter-sunday-2022"
              >
                Easter Info
              </mockConstructor>
            </li>
            <li>
              <mockConstructor
                className="button is-link is-size-4-tablet"
                to="/watch"
              >
                <span
                  className="icon is-medium"
                >
                  <FontAwesomeIcon
                    beat={false}
                    beatFade={false}
                    border={false}
                    bounce={false}
                    className=""
                    fade={false}
                    fixedWidth={false}
                    flip={null}
                    icon={
                      Array [
                        "fas",
                        "video",
                      ]
                    }
                    inverse={false}
                    listItem={false}
                    mask={null}
                    maskId={null}
                    pull={null}
                    pulse={false}
                    rotation={null}
                    shake={false}
                    size="1x"
                    spin={false}
                    swapOpacity={false}
                    symbol={false}
                    title=""
                    titleId={null}
                    transform={null}
                  />
                </span>
                <span>
                  Watch Messages
                </span>
              </mockConstructor>
            </li>
          </ul>
        </div>
      </nav>
    </div>
  </section>
  <section
    className="index-page verse-content section"
    dangerouslySetInnerHTML={
      Object {
        "__html": "<h2 id=\\"-come-to-me-a-you-who-are-wearyand-burdened-and-i-will-give-you-rest-\\">“Come to me, a you who are weary<br />and burdened, and I will give you rest.”</h2><p>Matthew 11:28 NIV</p>",
      }
    }
  />
  <section
    className="index-page about-us section"
  >
    <div
      className="columns container content"
    >
      <div
        className="column"
      >
        <FancyHeading
          className="has-text-centered"
          heading="Who We Are"
        />
      </div>
    </div>
    <div
      className="columns container is-multiline"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<h2 class=\\"container has-text-link is-1 title\\">We are people dedicated to inspiring others to discover hope in Jesus.</h2><div class=\\"column is-half-tablet is-one-third-desktop is-one-third-fullhd\\">
	<article class=\\"media px-5\\">
		<div class=\\"media-content\\">
			<h2 class=\\"title is-size-3 is-uppercase\\"><span class=\\"has-text-weight-light\\">Our</span> <span class=\\"decorated\\">Mission</span></h2>
			<p class=\\"content is-medium\\">We exist to inspire hope by helping people discover God’s purpose for their lives through a relationship with Jesus.</p>
		</div>
	</article>
</div><div class=\\"column is-half-tablet is-one-third-desktop is-one-third-fullhd\\">
	<article class=\\"media px-5\\">
		<div class=\\"media-content\\">
			<h2 class=\\"title is-size-3 is-uppercase\\"><span class=\\"has-text-weight-light\\">Our</span> <span class=\\"decorated\\">Strategy</span></h2>
			<p class=\\"content is-medium\\">We will do this by inspiring people to <em>follow</em> Jesus, <em>live</em> in community, <em>serve</em> those around them and <em>expand</em> God’s kingdom.</p>
		</div>
	</article>
</div><div class=\\"column is-half-tablet is-one-third-desktop is-one-third-fullhd\\">
	<article class=\\"media px-5\\">
		<div class=\\"media-content\\">
			<h2 class=\\"title is-size-3 is-uppercase\\"><span class=\\"has-text-weight-light\\">Our</span> <span class=\\"decorated\\">Values</span></h2>
			<p class=\\"content is-medium\\">Our values describe our passions and define the heart with which we will accomplish our mission and vision.</p>
		</div>
	</article>
</div>",
        }
      }
    />
    <div
      className="columns"
    >
      <div
        className="column is-full"
      >
        <div
          className="control has-text-centered my-3"
        >
          <mockConstructor
            className="button is-link has-text-weight-light is-medium"
            to="/about"
          >
            Learn More
          </mockConstructor>
        </div>
      </div>
    </div>
  </section>
  <section
    className="index-page gathering-section section px-0"
  >
    <div
      className="columns container content"
    >
      <div
        className="column"
      >
        <FancyHeading
          className="has-text-centered"
          heading="Weekly Gathering"
        />
      </div>
    </div>
    <div
      className="content"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<h2 id=\\"come-join-us-for-worshipand-so-much-more-\\">Come join us for worship<br />and so much more.</h2><figure class=\\"kg-card kg-image-card kg-width-full\\"><img src=\\"https://cms.inspiringhopechurch.com/content/images/2021/01/Image_Pathway_To_Hope_CROPPED_01.png\\" class=\\"kg-image\\" alt srcset=\\"https://cms.inspiringhopechurch.com/content/images/size/w600/2021/01/Image_Pathway_To_Hope_CROPPED_01.png 600w, https://cms.inspiringhopechurch.com/content/images/size/w1000/2021/01/Image_Pathway_To_Hope_CROPPED_01.png 1000w, https://cms.inspiringhopechurch.com/content/images/2021/01/Image_Pathway_To_Hope_CROPPED_01.png 1536w\\" /></figure><p><strong>SUNDAY MORNINGS AT 10:30 AM</strong></p><p>202 S Monument Ave, Hamilton, OH</p><p>Come join us as we seek to inspire each other in our walk with Jesus. We meet for worship most Sunday mornings, but occasionally take this time to serve our community.</p><p></p>",
        }
      }
    />
  </section>
  <section
    className="index-page caring-section section"
  >
    <div
      className="columns container content"
    >
      <div
        className="column is-full has-text-centered"
      >
        <h1>
          Caring for others
        </h1>
      </div>
    </div>
    <div
      className="container"
    >
      <div
        className="level is-mobile-mobile is-medium"
      >
        <div
          className="level-item has-text-centered"
        >
          <div>
            <figure
              className="heading is-inline-block"
            >
              <svg
                className="image is-96x96"
                viewBox="0 0 74.77 42.9"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Mask icon
                </title>
                <path
                  d="M73,1.23A4.71,4.71,0,0,0,68.75.51c-4,1.26-7.19,7.52-8.63,10.79-1.78-.7-17.67-6.89-22.77-6.89s-20.93,6.23-22.59,6.9C13.38,8.13,10.1,1.53,6,.23A4.74,4.74,0,0,0,1.71,1c-3.15,2.19-2,8.39,3.65,19A166.46,166.46,0,0,0,14.2,34.44v0l0,0,.06.08h0l.09.09a22.87,22.87,0,0,0,5.87,4.16A38.27,38.27,0,0,0,38.08,42.9c8.28,0,13.93-2.24,17.21-4.11a18.53,18.53,0,0,0,5.26-4.19l.14-.18v-.05l.05,0A161.7,161.7,0,0,0,69.53,20C75,9.54,76.19,3.41,73,1.23ZM20.21,17.1v-1a.58.58,0,0,1,.58-.59H53.67a.58.58,0,0,1,.58.59v1a.58.58,0,0,1-.58.58H20.79A.58.58,0,0,1,20.21,17.1Zm33.46,5a.58.58,0,0,1,.58.58v.88a.58.58,0,0,1-.58.58H20.79a.58.58,0,0,1-.58-.58v-.88a.58.58,0,0,1,.58-.58Zm-33.46,8v-.88a.58.58,0,0,1,.58-.58H53.67a.58.58,0,0,1,.58.58v.88a.58.58,0,0,1-.58.58H20.79A.58.58,0,0,1,20.21,30.06ZM6.57,19.27C1.56,9.83.11,3.73,2.49,2.08a3.39,3.39,0,0,1,3.1-.55C10.27,3,14.12,13.37,14.16,13.47h0V32C12.51,29.44,9.33,24.47,6.57,19.27Zm61.76.05c-2.78,5.24-6,10.25-7.64,12.72V13.49c.64-1.64,4.2-10.33,8.47-11.68a3.39,3.39,0,0,1,3.1.54C74.64,4,73.24,10,68.33,19.32Z"
                  fill="#fff"
                />
              </svg>
            </figure>
            <p
              className="is-uppercase"
            >
              Encouraging Masks
            </p>
          </div>
        </div>
        <div
          className="level-item has-text-centered"
        >
          <div>
            <figure
              className="heading is-inline-block"
            >
              <svg
                className="image is-96x96"
                viewBox="0 0 76.82 52.42"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Social distancing icon
                </title>
                <g
                  fill="#fff"
                >
                  <path
                    d="M45.7,24.85a1.07,1.07,0,0,0-1.51,1.51l1.2,1.2H31.28l1.2-1.2A1.07,1.07,0,0,0,31,24.85l-3,3a1.08,1.08,0,0,0,0,1.51l3,3a1.07,1.07,0,0,0,1.52-1.51l-1.2-1.21H45.39l-1.2,1.21a1.07,1.07,0,0,0,1.51,1.51l3-3a1.06,1.06,0,0,0,0-1.51Z"
                  />
                  <text
                    fontFamily="Roboto-Regular, Roboto, Roboto Light"
                    fontSize="9.68px"
                    letterSpacing="0.01em"
                    stroke="#fff"
                    strokeLinecap="round"
                    strokeLinejoin="round"
                    transform="translate(35.74 19.35)"
                  >
                    6
                  </text>
                  <circle
                    cx="63.56"
                    cy="8.96"
                    r="4.82"
                  />
                  <path
                    d="M73.86.91a2.4,2.4,0,0,0-3.78,3c2.17,2.74,2.57,6.19,1,8.68a7.21,7.21,0,0,1-3.93,2.81H60.11a2.4,2.4,0,0,0-1.6.57A14.47,14.47,0,0,0,56,18.51,18.92,18.92,0,0,0,52.24,30.8a2.4,2.4,0,0,0,2.41,2.33h.08a2.36,2.36,0,0,0,2.33-2.49,14.71,14.71,0,0,1,1.21-6.59A80.46,80.46,0,0,1,58,33l-.16,1.61V50a2.42,2.42,0,0,0,4.83,0V37.15h2V50a2.42,2.42,0,0,0,4.83,0V34.5l-.25-5.22c-.08-2.41-.08-4.91,0-7.48l.09-2.09a11.21,11.21,0,0,0,5.7-4.58A12.17,12.17,0,0,0,73.86.91Z"
                  />
                  <circle
                    cx="13.26"
                    cy="8.96"
                    r="4.82"
                  />
                  <path
                    d="M3,.91a2.4,2.4,0,1,1,3.77,3c-2.17,2.74-2.57,6.19-1,8.68a7.19,7.19,0,0,0,3.94,2.81h7.07a2.4,2.4,0,0,1,1.6.57,14.88,14.88,0,0,1,2.5,2.57A19,19,0,0,1,24.58,30.8a2.4,2.4,0,0,1-2.41,2.33h-.08a2.36,2.36,0,0,1-2.33-2.49,14.83,14.83,0,0,0-1.2-6.59A80.45,80.45,0,0,0,18.8,33L19,34.58V50a2.41,2.41,0,1,1-4.82,0V37.15h-2V50a2.41,2.41,0,1,1-4.82,0V34.5l.24-5.22c.08-2.41.08-4.91,0-7.48l-.08-2.09a11.19,11.19,0,0,1-5.71-4.58A12.16,12.16,0,0,1,3,.91Z"
                  />
                </g>
              </svg>
            </figure>
            <p
              className="is-uppercase"
            >
              Social Distancing
            </p>
          </div>
        </div>
        <div
          className="level-item has-text-centered"
        >
          <div>
            <figure
              className="heading is-inline-block"
            >
              <svg
                className="image is-96x96"
                viewBox="0 0 38.23 61.37"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Hand sanitizer icon
                </title>
                <g
                  fill="#fff"
                >
                  <path
                    d="M26.33,0H10.06a1.24,1.24,0,0,0-.64.18l-5,2.88a1.36,1.36,0,0,0-.64.81,1.37,1.37,0,0,0,1.3,1.7,1.45,1.45,0,0,0,.69-.18L10.47,2.7H26.33a1.33,1.33,0,0,0,1.34-1.35A1.33,1.33,0,0,0,26.33,0Z"
                  />
                  <rect
                    height="8.03"
                    width="5.82"
                    x="18.38"
                    y="4.04"
                  />
                  <path
                    d="M3.7,10.23C2.13,12.62,0,15.81,0,18.34A3.27,3.27,0,0,0,1.05,21a4,4,0,0,0,2.65.84A4,4,0,0,0,6.36,21a3.24,3.24,0,0,0,1-2.64C7.41,15.8,5.3,12.67,3.7,10.23Z"
                  />
                  <path
                    d="M18,13.44A2.51,2.51,0,0,0,15.51,16v1.85H27.08V16a2.51,2.51,0,0,0-2.53-2.53Z"
                  />
                  <path
                    d="M15.31,19.19A13.54,13.54,0,0,0,9,25.31a16.8,16.8,0,0,0-2,7v.05L4.36,58.08a3.28,3.28,0,0,0,3.31,3.29H34.92a3.28,3.28,0,0,0,3.31-3.29L35.65,32.35v0a17.62,17.62,0,0,0-2-7,13.4,13.4,0,0,0-6.29-6.12Z"
                  />
                </g>
              </svg>
            </figure>
            <p
              className="is-uppercase"
            >
              Providing Hand Sanitizer
            </p>
          </div>
        </div>
        <div
          className="level-item has-text-centered"
        >
          <div>
            <figure
              className="heading is-inline-block"
            >
              <svg
                className="image is-96x96"
                viewBox="0 0 55.98 75.53"
                xmlns="http://www.w3.org/2000/svg"
              >
                <title>
                  Disinfectant icon
                </title>
                <g
                  fill="#fff"
                >
                  <path
                    d="M47.44,50.19H3.09a.55.55,0,0,0-.55.56.57.57,0,0,0,0,.13l6,24.23a.56.56,0,0,0,.54.42H41.49a.56.56,0,0,0,.54-.42L48,50.88a.55.55,0,0,0-.41-.67Z"
                  />
                  <g
                    letterSpacing="0.01em"
                  >
                    <text
                      fontSize="24.55px"
                      fontWeight="500"
                      transform="translate(36.69 20.86)"
                    >
                      +
                    </text>
                    <text
                      fontSize="15.36px"
                      fontWeight="700"
                      transform="translate(46.95 29.82)"
                    >
                      +
                    </text>
                    <text
                      fontSize="15.36px"
                      fontWeight="700"
                      transform="translate(29.52 25.46)"
                    >
                      +
                    </text>
                  </g>
                  <path
                    d="M50,45.65H.55A.55.55,0,0,0,0,46.2v2.33a.56.56,0,0,0,.55.56h.77l.15,0a.65.65,0,0,1,.19,0H48.87a.58.58,0,0,1,.19,0l.15,0H50a.56.56,0,0,0,.55-.56V46.2A.55.55,0,0,0,50,45.65Z"
                  />
                  <path
                    d="M30.19,32.74H44.08a.55.55,0,0,0,.55-.55V28.5a.55.55,0,0,0-.55-.55H34.86a5.39,5.39,0,0,0-5.21,4.11.56.56,0,0,0,.42.67Z"
                  />
                  <rect
                    height="1.84"
                    rx="0.55"
                    width="6.27"
                    x="33.93"
                    y="33.85"
                  />
                  <path
                    d="M30.06,44.54h14a.55.55,0,0,0,.55-.55V41a4.24,4.24,0,0,0-4.24-4.24H33.75A4.24,4.24,0,0,0,29.51,41v3A.55.55,0,0,0,30.06,44.54Z"
                  />
                  <path
                    d="M7.93,39H22a.55.55,0,0,1,.55.55h0V44a.55.55,0,0,1-.55.55h-14A.54.54,0,0,1,7.38,44h0V39.56A.55.55,0,0,1,7.93,39Z"
                  />
                  <path
                    d="M8.89,37.9H21a.55.55,0,0,0,.55-.55.59.59,0,0,0-.09-.31L18.5,32.62a.56.56,0,0,0-.46-.25h-6.2a.54.54,0,0,0-.46.25L8.43,37a.55.55,0,0,0,.46.86Z"
                  />
                  <rect
                    height="6.27"
                    rx="0.55"
                    width="6.27"
                    x="11.8"
                    y="25"
                  />
                </g>
              </svg>
            </figure>
            <p
              className="is-uppercase"
            >
              Maintaining Clean Spaces
            </p>
          </div>
        </div>
      </div>
    </div>
  </section>
  <section
    className="index-page kids-section section"
  >
    <div
      className="columns container"
      dangerouslySetInnerHTML={
        Object {
          "__html": "<figure class=\\"kg-card kg-image-card\\"><img src=\\"https://cms.inspiringhopechurch.com/content/images/2021/01/Logos_Inspire_Kids_Badge_01.svg\\" class=\\"kg-image\\" alt /></figure><p><strong>Inspire Kids</strong></p><p>Inspire Kids is designed for infants through fifth grade students. Our team of qualified volunteers work hard to ensure that your<br />kids will love Sunday morning gathering!</p>",
        }
      }
    />
    <div
      className="columns container"
    >
      <div
        className="column is-full"
      >
        <div
          className="control has-text-centered my-3"
        >
          <mockConstructor
            className="button is-link has-text-weight-light is-medium"
            to="/get-connected"
          >
            Get Connected
          </mockConstructor>
        </div>
      </div>
    </div>
  </section>
  <section
    className="index-page blog-section section"
  >
    <div
      className="columns content"
    >
      <div
        className="column"
      >
        <FancyHeading
          className="has-text-centered"
          heading="Inspiring Moments"
        />
      </div>
    </div>
    <div
      className="columns is-multiline is-centered"
    >
      <BlogItem
        blogAuthor="Inspiring Hope Church"
        blogDate="01 June, 2020"
        blogExcerpt="The summer is off to a great start! Even though it is definitely different this
year, we have taken advantage of our state opening up some. We've made some more
connections by meeting more neighbors, talking and working with local pastors on
the same mission, and really trying to get involved in the community. Our parks
have opened back up so we've visited the playgrounds a couple of times. We are
planning on doing some community outreaches this next month by providing our
local community worker"
        blogImageObj={
          Object {
            "backgroundColor": "#080808",
            "height": 788.0000000000001,
            "images": Object {
              "fallback": Object {
                "sizes": "(min-width: 940px) 940px, 100vw",
                "src": "/static/0209a02eeede1cb3592918c9abd69fd3/65bd1/Add-a-little-bit-of-body-text.png",
                "srcSet": "/static/0209a02eeede1cb3592918c9abd69fd3/c1bf5/Add-a-little-bit-of-body-text.png 235w,
/static/0209a02eeede1cb3592918c9abd69fd3/dd1c7/Add-a-little-bit-of-body-text.png 470w,
/static/0209a02eeede1cb3592918c9abd69fd3/65bd1/Add-a-little-bit-of-body-text.png 940w",
              },
              "sources": Array [
                Object {
                  "sizes": "(min-width: 940px) 940px, 100vw",
                  "srcSet": "/static/0209a02eeede1cb3592918c9abd69fd3/60ae1/Add-a-little-bit-of-body-text.webp 235w,
/static/0209a02eeede1cb3592918c9abd69fd3/4fcad/Add-a-little-bit-of-body-text.webp 470w,
/static/0209a02eeede1cb3592918c9abd69fd3/dfb29/Add-a-little-bit-of-body-text.webp 940w",
                  "type": "image/webp",
                },
              ],
            },
            "layout": "constrained",
            "width": 940,
          }
        }
        blogLink="/blog/our-church-planting-journey-volume-6"
        blogReadingTime={2}
        blogTitle="Our Church Planting Journey: Volume 6"
        key="Ghost__Post__5f4aba3168998602af191113"
        onBlogIndex={false}
      />
      <BlogItem
        blogAuthor="Inspiring Hope Church"
        blogDate="16 May, 2020"
        blogExcerpt="Update...
The last couple of months have been so different than we had expected them to
be. Even in the midst of different, we have found an overwhelming peace with
where God has us and what He is doing. We have clung to our faith and the things
we could control and we prayed and let go and let God work in the ways we
couldn’t. In April we celebrated Ben’s birthday by surprising him with a zoom
call with many of his friends to wish him a happy birthday. Thank you to all of
you who were able to b"
        blogImageObj={
          Object {
            "backgroundColor": "#080808",
            "height": 788.0000000000001,
            "images": Object {
              "fallback": Object {
                "sizes": "(min-width: 940px) 940px, 100vw",
                "src": "/static/0209a02eeede1cb3592918c9abd69fd3/65bd1/Add-a-little-bit-of-body-text.png",
                "srcSet": "/static/0209a02eeede1cb3592918c9abd69fd3/c1bf5/Add-a-little-bit-of-body-text.png 235w,
/static/0209a02eeede1cb3592918c9abd69fd3/dd1c7/Add-a-little-bit-of-body-text.png 470w,
/static/0209a02eeede1cb3592918c9abd69fd3/65bd1/Add-a-little-bit-of-body-text.png 940w",
              },
              "sources": Array [
                Object {
                  "sizes": "(min-width: 940px) 940px, 100vw",
                  "srcSet": "/static/0209a02eeede1cb3592918c9abd69fd3/60ae1/Add-a-little-bit-of-body-text.webp 235w,
/static/0209a02eeede1cb3592918c9abd69fd3/4fcad/Add-a-little-bit-of-body-text.webp 470w,
/static/0209a02eeede1cb3592918c9abd69fd3/dfb29/Add-a-little-bit-of-body-text.webp 940w",
                  "type": "image/webp",
                },
              ],
            },
            "layout": "constrained",
            "width": 940,
          }
        }
        blogLink="/blog/our-church-planting-journey-volume-5"
        blogReadingTime={3}
        blogTitle="Our Church Planting Journey: Volume 5"
        key="Ghost__Post__5f4ab4bc68998602af1910dc"
        onBlogIndex={false}
      />
    </div>
    <div
      className="columns container"
    >
      <div
        className="column is-full"
      >
        <div
          className="control has-text-centered my-3"
        >
          <mockConstructor
            className="button is-link has-text-weight-light is-medium"
            to="/blog"
          >
            More Posts
          </mockConstructor>
        </div>
      </div>
    </div>
  </section>
  <section
    className="index-page subscriptions  is-radiusless is-clipped mb-0"
  >
    <div
      className="columns is-multiline is-mobile is-centered is-vcentered has-text-centered"
    >
      <div
        className="column container is-fluid is-narrow is-full"
      >
        <label
          className="content"
          htmlFor="subscribe"
        >
          Get Inspiring Hope's latest updates.
        </label>
      </div>
      <div
        className="column is-narrow"
      >
        <form>
          <div
            className="field has-addons"
          >
            <div
              className="control"
            >
              <input
                className="input is-link"
                id="subscribe"
                name="subscribe"
                onChange={[Function]}
                placeholder="Sign up for newsletter"
                type="email"
                value=""
              />
            </div>
            <div
              className="control"
            >
              <button
                className="button is-link "
                data-testid="submit-button"
                onClick={[Function]}
                type="submit"
              >
                Sign Up
              </button>
            </div>
          </div>
        </form>
      </div>
    </div>
  </section>
</Fragment>
`;
